include/deal.II/base/qprojector.h

```diff
@@ -87,19 +89,40 @@ class QProjector
    * is used on face <tt>face_no</tt>. For further details, see the general
    * doc for this class.
    */
-  static void
+  DEAL_II_DEPRECATED static void
```

Comment:
Can you add a statement in the function documentation suggesting that the function of the same name with a `ReferenceCell::Type` argument should be used?

Reply:
Done!